### DISCUSSION TITLE
Future Ready Talent not responding

### DISCUSSION THREAD

**Discussion by SagnikPahari** [github/india#282] [Students]
Cannot update my details in the future ready talent site. The site is not responding. Anyone else having similar issues?

**Comment by RomaJadhav** [comment]
> Cannot update my details in the future ready talent site. The site is not responding. Anyone else having similar issues?

Same was happening for me.. do this-->
just clear your browsing data on your browser
Then open the login page in incognito mode and fill up everything again.

**Reply by SagnikPahari** [reply]
thanks. it worked

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
